app/org/sagebionetworks/bridge/dynamodb/DynamoStudyConsent1.java

```diff
@@ -18,20 +18,20 @@
 @DynamoDBTable(tableName = "StudyConsent1")
 public final class DynamoStudyConsent1 implements StudyConsent {
 
-    private String studyKey;
+    private String subpopGuid;
     private long createdOn;
     private boolean active;
     private String storagePath;
     private Long version;
 
     @Override
-    @DynamoDBHashKey
+    @DynamoDBHashKey(attributeName="studyKey")
     @JsonIgnore
```

Comment:
I'm removing the JsonIgnore annotation, and added a test to ensure StudyConsent serializes correctly because it's exposed in one API call where we return a list of consents. It looks the same as StudyConsentView, but doesn't have the contents of the consent (which is now documented).